src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/integration/StaffIntegrationTest.kt

```diff
@@ -11,58 +11,92 @@ import org.springframework.http.MediaType
 import org.springframework.test.context.jdbc.Sql
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity.CommunityOffenderManager
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity.PrisonUser
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity.Staff
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.integration.wiremock.DeliusMockServer
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.ComReviewCount
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.UpdateComRequest
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.UpdatePrisonUserRequest
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.repository.StaffRepository
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.StaffKind
 import java.util.concurrent.CountDownLatch
 import java.util.concurrent.ExecutionException
 import java.util.concurrent.Executors
 import java.util.concurrent.Future
 
 class StaffIntegrationTest : IntegrationTestBase() {
+
   @Autowired
   lateinit var staffRepository: StaffRepository
 
   @Test
-  @Sql(
-    "classpath:test_data/clear-all-data.sql",
-  )
-  fun `Add and update a COM record`() {
-    doUpdate("/com/update", updateCom)
+  fun `Add a COM record`() {
+    // Given
+    val request = UpdateComRequest(
+      staffIdentifier = 2001L,
+      staffUsername = "newbloggs",
+      staffEmail = "newbloggs@probation.gov.uk",
+      firstName = "New",
+      lastName = "NewBloggs",
+    )
 
-    assertThat(staffRepository.count()).isEqualTo(1)
+    // When
+    doUpdate("/com/update", request)
 
-    with(staffRepository.findAll().first() as CommunityOffenderManager) {
-      assertThat(staffIdentifier).isEqualTo(updateCom.staffIdentifier)
-      assertThat(username).isEqualTo(updateCom.staffUsername.uppercase())
-      assertThat(email).isEqualTo(updateCom.staffEmail)
-      assertThat(firstName).isEqualTo(updateCom.firstName)
-      assertThat(lastName).isEqualTo(updateCom.lastName)
-    }
+    // Then
+    val staff = staffRepository.findById(9)
```

Comment:
Be good for this to be more intention revealing.. it's not very obvious that a staff record has been added if we're just checking the state of a given staff record with a specific id